prime-router/src/main/kotlin/DetailedSubmissionHistory.kt

```diff
@@ -298,50 +255,11 @@ class DetailedSubmissionHistory(
         actualCompletionAt = calculateActualCompletionAt(realDestinations)
     }
 
-    /**
-     * Consolidate the [logs] filtered by an optional [filterBy] action level, so to list similar messages once
-     * with a list of items they relate to.
-     * @return the consolidated list of logs
-     */
-    internal fun consolidateLogs(filterBy: ActionLogLevel? = null):
-        List<ConsolidatedActionLog> {
-        val consolidatedList = mutableListOf<ConsolidatedActionLog>()
-
-        // First filter the logs and sort by the message.  This first sorting can take care of sorting old messages
-        // that contain index numbers like "Report 3: xxxx"
-        val filteredList = when (filterBy) {
-            null -> logs
-            else -> logs.filter { it.type == filterBy }
-        }.sortedBy { it.detail.message }
-        // Now order the list so that logs contain first non-item messages, then item messages, and item messages
-        // are sorted by index.
-        val orderedList = (
-            filteredList.filter { it.scope != ActionLogScope.item }.sortedBy { it.scope } +
-                filteredList.filter { it.scope == ActionLogScope.item }.sortedBy { it.index }
-            ).toMutableList()
-        // Now consolidate the list
-        while (orderedList.isNotEmpty()) {
-            // Grab the first log.
-            val consolidatedLog = ConsolidatedActionLog(orderedList.first())
-            orderedList.removeFirst()
-            // Now find similar logs and consolidate it.  Note by using the iterator we can delete on the fly.
-            with(orderedList.iterator()) {
-                forEach { log ->
-                    if (consolidatedLog.canBeConsolidatedWith(log)) {
-                        consolidatedLog.add(log)
-                        remove()
-                    }
-                }
-            }
-            consolidatedList.add(consolidatedLog)
-        }
-
-        return consolidatedList
-    }
-
     /**
      * Runs the calculations for the overallStatus field so that it can be done during init.
-     * @returns The status from the Status enum that matches the current Submission state.
+     *
+     * @param realDestinations destinations where items have gone through and thus should be calculated
```

Comment:
For functions, try to use [] when you can.  This was the preference stated a while back by the entire RS team.  See kdoc documentation.  Do for the other functions too.